mllib/src/test/scala/org/apache/spark/ml/feature/BucketedRandomProjectionLSHSuite.scala

```diff
@@ -98,6 +97,21 @@ class BucketedRandomProjectionLSHSuite
     MLTestingUtils.checkCopyAndUids(brp, brpModel)
   }
 
+  test("BucketedRandomProjectionLSH: streaming transform") {
+    val brp = new BucketedRandomProjectionLSH()
+      .setNumHashTables(2)
+      .setInputCol("keys")
+      .setOutputCol("values")
+      .setBucketLength(1.0)
+      .setSeed(12345)
+    val brpModel = brp.fit(dataset)
+
+    testTransformer[Tuple1[Vector]](dataset.toDF(), brpModel, "keys", "values") {
+      case Row(_: Vector, values: Seq[_]) =>
```

Comment:
So here do not need to select "keys" column ?

Reply:
I don't think we have to.  The main thing here is to make sure that the transform really does happen.  Other tests check validity of the values.

Reply:
OK. so I prefer to use simpler code:
```
testTransformer[Tuple1[Vector]](dataset.toDF(), brpModel, "values") {
    case Row(values: Seq[_]) =>
...
```